examples/pipelines/simple_pipeline/components/image_cropping/fondant_component.yaml

```diff
@@ -0,0 +1,30 @@
+name: Image cropping
+description: Component that removes single-colored borders around images and crops them appropriately
+image: ghcr.io/ml6team/image_cropping:latest
+
+input_subsets:
+  images:
+    fields:
+      data:
+        type: binary
+
+output_subsets:
+  images_crop:
```

Comment:
I think we can overwrite the original `images` subset.

Reply:
done!